.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,50 @@
+name: 🚀 Feature Request
+description: Suggest an idea or a feature for this project
+labels: [ ":seedling: feature" ]
```

Comment:
This is a proposal for a new label to be created before this PR gets merged.

Reply:
We have `enhancement` and `bug` currently, can we use those instead?

Reply:
Also... I have a deep animosity to emojis in commits & code, so can we do a compromise and keep the emoji in the title but not in the label? :)

Reply:
@rhamzeh Absolutely, we don't really need to create new ones, we can reuse current ones, or even update them. 

I'm just biased towards using emojis 😄 , here's an example in action: https://github.com/auth0/terraform-provider-auth0/labels. 

I can revert the label at any time, just waiting for further feedback to apply more changes in bulk.

Reply:
Removed in [`8b59dc8` (#1262)](https://github.com/openfga/openfga/pull/1262/commits/8b59dc8d3ff120b7d0e8cb521b4101cfc5e90df8).